Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.cpp

```diff
@@ -111,7 +111,7 @@ bool SQLiteStorageArea::isEmpty()
 
     auto statement = cachedStatement(StatementType::CountItems);
     if (!statement || statement->step() != SQLITE_ROW) {
-        RELEASE_LOG_ERROR(Storage, "SQLiteStorageArea::isEmpty failed on executing statement (%d) - %{%public}s", m_database->lastError(), m_database->lastErrorMsg());
+        RELEASE_LOG_ERROR(Storage, "SQLiteStorageArea::isEmpty failed on executing statement (%d) - %"  PUBLIC_LOG_STRING, m_database->lastError(), m_database->lastErrorMsg());
```

Comment:
It looks like the extra percent sign in `%{%public}s` was a typo, not sure how it behave.